Main.java

```diff
@@ -77,47 +69,19 @@ private static void wordCount(String textFilePath, String searchKeyword) {
                 }
             }
             if (count == 0) {
-                errorMessage = "Word does not exist";
-                System.out.println(errorMessage);
-                Database.databaseOperation(textFilePath, searchKeyword, searchResult = "Failure", errorMessage, count);
+                Constants.errorMessage5 = "Word not found";

```

Comment:
You have already defined these messages as constants, then why are you reassigning them?

Reply:
Mistake from my part. Will remove them immediately.